pallets/avn-transaction-payment/src/fee_adjustment_config.rs

```diff
@@ -180,7 +180,7 @@ impl<T: Config> FixedFeeConfig<T> {
 pub struct PercentageFeeConfig<T: Config> {
     pub percentage: u32,
     #[codec(skip)]
-    _marker: PhantomData<T>,
+    pub _marker: PhantomData<T>,
```

Comment:
If you don't want to expose this publicly, you can create a `new` function like this:
```
pub fn new(percentage: u32) -> Self {        
        return PercentageFeeConfig::<T> { percentage, _marker:  PhantomData::<T>}
    }
```
and call it when you want to create a new instance of this struct